build-machinery/make/targets/openshift/operator/release.mk

```diff
@@ -0,0 +1,7 @@
+# If we need unified behaviour specific to operators, this folder is the place.
+
+# It seems that our previous origin-release jq based replacement is suppose to be done
+# with `oc adm release new` so it might drop this target.
```

Comment:
Please add it until the alternative is available. I use that target on a daily basis.

Reply:
Yeah I wanted to talk to you about that but didn't want to bother you when you were busy. Let's sync tomorrow.

Reply:
Also this is not dropping it, the end makefiles can still define it in the interim.

Reply:
ok, to leave it around till we decide about the new/old way and keep it in the end makefiles?